src/backends/backend/triton_model.cc

```diff
@@ -157,9 +157,12 @@ TritonModel::Create(
   }
   local_model->initialized_ = true;
 
+  // FIXME: Should we use device blocking even for the sequential models?
```

Comment:
@GuanLuo In our current TensorRT backend we have separate runner per instance if model is sequential. Do you see it being a problem for DEVICE_BLOCKING policy? Should this execution policy be ignored if a sequential model is detected?

Reply:
I would take a closer look.

It should be fine as long as it is doing the right thing for sequence model, originally the sequence batcher setup was kept the same as other framework just because it would require a re-work on the sequence batcher otherwise and we didn't care too much for device blocking on sequence case.

Since you are re-working the batchers, you should be able to make it right.